examples/plotting/file/sparkline.py

```diff
@@ -0,0 +1,73 @@
+import pandas as pd
+from bokeh.charts import TimeSeries
+from bokeh.models import (
+    Legend,
+    Circle,
+    HoverTool,
+    ColumnDataSource,
+    GlyphRenderer
+)
+from bokeh.palettes import brewer
+
+TOOLS = "hover"
+palette = brewer["Blues"][3]
+
+apple = pd.read_csv(
+    "http://ichart.yahoo.com/table.csv?s=AAPL&a=0&b=1&c=2000&d=0&e=1&f=2010",
+    parse_dates=['Date']
+)
+
+# Basic sparkline
+
+sparkline = TimeSeries(
+    apple[['Adj Close', 'Date']], index='Date',
+    title="", tools=TOOLS, height=100, palette=palette,
+    ylabel='', xlabel='', xgrid=None, ygrid=None,
+    filename="sparkline.html"
+)
+sparkline.add_layout(
+    Legend(
+        legends=[('Apple', None), ],
+        orientation="top_left",
+        border_line_color="white",
+        label_text_font_style="bold"
+    )
+)
+
+sparkline.left[0].visible = False
+sparkline.below[0].visible = False
+sparkline.toolbar_location = None
+sparkline.min_border = 0
+sparkline.outline_line_color = None
+
+# Optional goodies
+# - Add a dot at either end
+# - with hover information
+fmt = '%d %b \'%y'
+apple['FormattedDate'] = apple['Date'].apply(lambda x: x.strftime(fmt))
```

Comment:
Sadly, with `@name{format}` syntax you can't format dates. Hopefully this will be resolved in #1879.